apps/artemis-web/components/status-table/status-table.tsx

```diff
@@ -27,7 +52,14 @@ class StatusTable extends Component<StatsType, {}> {
             STATS_DATA.view_processes.map((process, i) => {
               return (
                 <tr key={i}>
-                  <td>{process.name}</td>
+                  <td
+                    data-toggle="tooltip"
+                    data-placement="top"
+                    title={tooltips[process.name]}
```

Comment:
if this doesn't exist then we don't have an error right?

Reply:
If its not existent, I now assign an empty string to the tooltip.